generator/epic.go

```diff
@@ -189,23 +80,17 @@ func CreateEpicFile(stateFields []*gp.FieldDescriptorProto, customFields []*gp.F
 		repeated := field.GetLabel() == 3
 
 		// verify the method annotations
-		methods, err := GetFieldOptionsString(field, state.E_Method)
+    fieldAnnotations, err := GetFieldOptions(field)
 		if err != nil {
 			return nil, fmt.Errorf("Error getting field level annotations: %v", err)
 		}
 
 		// field level overrides for timeout/retry
-		timeout, err := GetFieldAnnotationInt(field, state.E_Timeout)
-		if err != nil {
-			return nil, fmt.Errorf("Error getting field level timeout annotation: %v", err)
-		}
+    timeout := fieldAnnotations.GetTimeout()
 		if timeout == -1 { // if it wasn't overriden
```

Comment:
I'm not sure these integer annotations return -1 if not set anymore. That was the value assigned in GetFieldOptions, but those methods are more generic now.